### PR TITLE
Added Pagination for Products

### PR DESCRIPTION
**Pull request by jaypowar00**
### Added Pagination for Products queries

- added feature :
- [x] pagination for all products results

### CODE REVIEW

**Comment by mrwhoknows55** [comment]
@jaypowar00 pull changes from dev and push

File: ecommerce_api/settings.py [hunk 1]
```diff
@@ -26,20 +28,16 @@
 DEBUG = True
 
 ALLOWED_HOSTS = [
-    'http://127.0.0.1:3000',
```
Comment (mrwhoknows55) [inline ecommerce_api/settings.py:29]: @jaypowar00 remind me to discuss about this

File: ecommerce_api/settings.py [hunk 2]
```diff
@@ -135,40 +133,31 @@
...
 
+
 CSRF_COOKIE_DOMAIN = 'ecmrce-suflowapi.herokuapp.com'
 CSRF_COOKIE_SECURE = True
 CSRF_COOKIE_SAMESITE = 'None'
 SESSION_COOKIE_SAMESITE = 'None'
 SESSION_COOKIE_SAMESITE_FORCE_ALL = True
 SESSION_COOKIE_SECURE = True
 CSRF_TRUSTED_ORIGINS = [
-    'http://127.0.0.1:5000',
```
Comment (mrwhoknows55) [inline ecommerce_api/settings.py:145]: same as above here

**Comment by jaypowar00** [comment]
> @jaypowar00 pull changes from dev and push

@mrwhoknows55 Done 👍

**Pull request merged**